solutions/usaco-639.mdx

```diff
@@ -50,4 +50,31 @@ int main(){
 
 </CPPSection>
 
+<PySection>
+
+```py
+fin = open('diamond.in', 'r')
+
+N, K = map(int, fin.readline().split())
+
+diamonds = []
+
+for _ in range(N):
+	diamonds.append(int(fin.readline()))
+
+tracker = []
```

Comment:
```suggestion
```
this isn't used anywhere, did you forget to remove?

Reply:
oops, sorry about that.